setup.py

```diff
@@ -28,6 +28,7 @@
 # 'Development Status :: 5 - Production/Stable'``
 release_status = "Development Status :: 4 - Beta"
 dependencies = [
+    "bigframes >= 1.17.0",
```

Comment:
Please don't.

In the design I shared that this should be an optional dependency. That means it should be an "extra".

Reply:
Ack. Moved it to extra.